From 1748568978ea6701375a862120b3158e1bc63680 Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Sun, 12 Feb 2023 16:58:43 +0200
Subject: [PATCH] [SW-120178] fwupd: move comment to correct place

The comment about handling the commands state was kept in the wrong
place where is does less sense.
Move the comment to a better place where it actually makes sense.
No functional changes.

Change-Id: I83acc52b0088ed9e1a5a890e01ad3c025a796ec3
Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/lib/preboot/module_fwupd.c b/lib/preboot/module_fwupd.c
index 14730fc..65762ad 100644
--- a/lib/preboot/module_fwupd.c
+++ b/lib/preboot/module_fwupd.c
@@ -565,7 +565,6 @@
 	case HL_UPD_GET_PROTO_STATUS:
 		cmd_str = "proto_status";
 		module_fwupd_send_status(state, HL_UPD_STS_ACK, NULL);
-		/* We ACK the command, but handle only on CLR, so save cmd */
 		break;
 	default:
 		cmd_str = "unknown";
@@ -574,7 +573,7 @@
 		break;
 	}
 
-	if (!err)
+	if (!err) /* We ACK the command, but handle only on CLR, so save cmd */
 		module_fwupd_state_shift(state);
 
 	if (state->cmd.cmd != HL_UPD_CLR_STS) {
